pkl-core/src/main/java/org/pkl/core/runtime/TestRunner.java

```diff
@@ -387,74 +388,85 @@ private static String getDisplayUri(ModuleInfo moduleInfo) {
   }
 
   private static Failure factFailure(SourceSection sourceSection, String location) {
-    return new Failure("Fact Failure", sourceSection.getCharacters().toString(), location);
+    String message = sourceSection.getCharacters().toString() + " " + renderLocation(location);
+    return new Failure("Fact Failure", message);
   }
 
   private static Failure exampleLengthMismatchFailure(
       String location, String property, int expectedLength, int actualLength) {
     String msg =
-        "Output mismatch: Expected \""
+        renderLocation(location)
+            + "\n"
```

Comment:
I think this is the extra new-line I mentioned before that I would remove

Reply:
Yup; this is stripped by `stripFileAndLines`